exo.jcr.component.core/src/main/java/org/exoplatform/services/jcr/impl/dataflow/SpoolConfig.java

```diff
@@ -37,8 +38,43 @@
 
    public File tempDirectory = new File(PropertyManager.getProperty("java.io.tmpdir"));
 
+   private static final String  FORCE_CLEAN_SWAP = "exo.jcr.spoolConfig.force.clean.swap";
+
+   private static final String  FORCE_CLEAN_SWAP_LIVE_TIME = "exo.jcr.spoolConfig.swap.live.time";
+
    public int maxBufferSize = WorkspaceDataContainer.DEF_MAXBUFFERSIZE;
 
+
+   private static Map<String, SpoolConfig> spoolConfigList = new HashMap<String, SpoolConfig>();
+
+   public static final boolean forceClean;
+
+   public static int liveTime = -1 ;
+
+   static
+   {
+      String enable = PropertyManager.getProperty(FORCE_CLEAN_SWAP);
+      boolean value = false;
+      if (enable!= null && !enable.isEmpty())
+      {
+         value = Boolean.valueOf(enable);
+      }
+      forceClean = value;
+
+      if (forceClean)
+      {
+         try
+         {
+            liveTime = Integer.parseInt(System.getProperty(FORCE_CLEAN_SWAP_LIVE_TIME));
+
+         }
+         catch (NumberFormatException nex)
+         {
+            liveTime = -1;
```

Comment:
We should add a log here (warn) to let the customer know that its setting is not correct.